assets/customer.css

```diff
@@ -113,31 +116,22 @@
     padding-right: 2.2rem;
   }
 
-  .customer thead th,
-  .customer tbody td:first-of-type {
+  .customer thead th {
     padding-top: 2.4rem;
     padding-bottom: 2.4rem;
   }
-}
 
-.customer thead th {
-  font-size: 1.2rem;
-  letter-spacing: 0.07rem;
-  text-transform: uppercase;
-}
-
-@media screen and (min-width: 750px) {
   .customer th:first-of-type,
   .customer td:first-of-type {
     padding-left: 2.2rem;
   }
-}
 
-.customer tbody td:first-of-type {
-  padding-top: 4rem;
-}
+  .customer tbody td {
```

Comment:
small nitpick, you could just add this selector above so you don't have to rewrite the same padding value. 
```css
.customer thead th,
.customer tbody td {
    padding-top: 2.4rem;
    padding-bottom: 2.4rem;
  }

Reply:
Good one, I made the change.